qemu/tests/multi_vms_with_stress.py

```diff
@@ -95,6 +96,14 @@ def wait_for_shutdown_all_vms(vms, sessions):
     run_stress_background()
     is_stress_alive()
 
+    if params.get('set_maxcpus', 'no') == 'yes':
```

Comment:
fyi: params.get_boolean('set_maxcpus'), the code is OK for me

Reply:
thanks a lot, correct later